Dockerfiles/ci/run_tests.py

```diff
@@ -7,33 +7,82 @@
+
+    def test_validate_deprecated_operator_bundle_failure(self):
+        operator_work_dir = "{}/example-bundle-default-deprecated".format(self.test_dir)
+        work_dir = "{}/example-workdir-v45-v49".format(self.test_dir)
+        exec_cmd = "ansible-playbook -vvv -i localhost, --connection local \
+                operator-test-playbooks/validate-operator-bundle.yml \
+                -e 'operator_work_dir={operator_work_dir}' \
+                -e 'work_dir={work_dir}'".format(operator_work_dir=operator_work_dir,
+                                                 work_dir=work_dir)
+        playbook_command = subprocess.run(exec_cmd, shell=True)
+
+        print(playbook_command)
+        self.assertTrue(playbook_command.returncode == 0)
+        self.assertTrue(path.exists("{}/validation-rc.txt".format(work_dir)))
+        self.assertTrue(path.exists("{}/validation-output.txt".format(work_dir)))
+        with open("{}/validation-rc.txt".format(work_dir), "r") as fd:
+            validation_rc = fd.read()
+            print(validation_rc)
+            self.assertEqual(validation_rc, "1")
+
+    def test_validate_v48_operator_bundle_success(self):
+        operator_work_dir = "{}/example-bundle-default-deprecated".format(self.test_dir)
+        work_dir = "{}/example-workdir-v45-v48".format(self.test_dir)
+        exec_cmd = "ansible-playbook -vvv -i localhost, --connection local \
+                operator-test-playbooks/validate-operator-bundle.yml \
+                -e 'operator_work_dir={operator_work_dir}' \
+                -e 'work_dir={work_dir}'".format(operator_work_dir=operator_work_dir,
+                                                 work_dir=work_dir)
+        playbook_command = subprocess.run(exec_cmd, shell=True)
+
+        print(playbook_command)
+        self.assertTrue(playbook_command.returncode == 0)
+        self.assertTrue(path.exists("{}/validation-rc.txt".format(work_dir)))
+        self.assertTrue(path.exists("{}/validation-output.txt".format(work_dir)))
+        with open("{}/validation-rc.txt".format(work_dir), "r") as fd:
+            validation_rc = fd.read()
+            print(validation_rc)
+            self.assertEqual(validation_rc, "0")
+
+    def test_validate_v43_operator_bundle_failure(self):
+        operator_work_dir = "{}/example-bundle-default-positive".format(self.test_dir)
+        work_dir = "{}/example-workdir-v43-v44".format(self.test_dir)
+        exec_cmd = "ansible-playbook -vvv -i localhost, --connection local \
+                operator-test-playbooks/validate-operator-bundle.yml \
+                -e 'operator_work_dir={operator_work_dir}' \
+                -e 'work_dir={work_dir}'".format(operator_work_dir=operator_work_dir,
+                                                 work_dir=work_dir)
+        playbook_command = subprocess.run(exec_cmd, shell=True)
+        print(playbook_command)
+        self.assertTrue(playbook_command.returncode == 2)
```

Comment:
Please add a check to expected error message in the output as addition to checking return code.

Reply:
Ack, added the check for the error message.